### DISCUSSION TITLE
Reaching out for Mentors GSoC: OpenVINO Extension for Automatic1111 Stable Diffusion WebUI

### DISCUSSION THREAD

**Discussion by mengbingrock** [openvinotoolkit/openvino#23594] [Google Summer of Code]
My last internship is about accelerating Diffusion models in a startup in LLM. I've done similar jobs with TensorRT. Hope to work with you guys and mentors on OpenVINO this time

**Comment by yury-gorbachev** [comment]
@ynimmaga, FYI

**Comment by likholat** [comment]
Hi @mengbingrock ! Thank you for your interest in the project. 
You should start with [Prerequisite Task](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#prerequisite-task). Please take any task form the [board](https://github.com/orgs/openvinotoolkit/projects/3/views/1).

**Reply by likholat** [reply]
@mengbingrock, sorry, my bad - found your PRs for openvino and openvino.genai. 
Thanks for submitting your proposal!

**Reply by mengbingrock** [reply]
Nice to meet you @likholat !  In case anything needs improvement in my proposal, you could comment anytime here and I'll modify it accordingly.  I wish I could have a chance to work for OpenVINO this summer!

**Reply by mengbingrock** [reply]
This is the my previous work to compile stable diffusion controlnet to TensorRT.  [code](https://github.com/mengbingrock/diffusion-tensorRT/commit/1e0706e22801dc85da7b6fcc9ff4900f2ab06365)

The code integrating to gradio is going to added soon. I hope this time I could enable OpenVINO as the backend.

**Comment by mengbingrock** [comment]
Dear mentors @likholat and @cavusmustafa, Please accept my sincere thanks for giving me the opportunity for this project! I'll begin this project this week and have my current code and doc ready.

**Reply by likholat** [reply]
Hi @mengbingrock, congrats! I'm really happy to see you in GSoC 2024!
Please, check your email - I'm excited to schedule our first call and start the project!
